### ISSUE TITLE
New root node missing icon

### ISSUE THREAD

**Issue by forworkdelete** [mbraak/jqTree#756]
I've been playing around with the jqTree plugin in a brand new asp.net razor application (.NET framework 4.7.2) and I've come across something strange. When I add a new root node using prependNode, the left icon is missing. I've added custom icons onCreateLi and those show up as expected. Is there additional setup that I've missed? To add, I've observed this behavior when I drag and drop a file (child node) outside the parent folder (root node). My custom icons are added but it's missing the leftmost icon.   
I've been referencing the documentation pretty extensively, and everything else has been fairly straight forward. 

Video demo: 
https://github.com/mbraak/jqTree/assets/142816876/734b8784-358c-4302-b598-6ea0ececb4fc

Relevant code snippets: 
```
<body>
<button id="addFolder" class="btn btn-primary" onclick="createFolder()">Add Folder</button> 

<div id="tree1"></div>

<script type="text/javascript">
        function createFolder() {
            var id = 25;

            $('#tree1').tree(
                'prependNode',
                {
                    name: 'NewFolder',
                    id: id,
                    nodeType: 'folder'
                }
            ); 
        }
</script>
<!--config/tree setup-->
<script type="text/javascript">
$('#tree1').tree({
    data: data, //static data js array (not from server)
    dragAndDrop: true,
    closedIcon: $('<i class="fa-solid fa-folder"></i>'),
    openedIcon: $('<i class="fa-solid fa-folder-open"></i>'),
    showEmptyFolder: true,
    onCreateLi: function (node, $li) {
        var $title = $li.find('.jqtree-title');
        // common functions
        $title.append($('<button class="btn" onclick="downloadFile(this)"><i class="fa-solid fa-download px-2"></i></button>')); //download
        $title.append($('<button class="btn" data-bs-toggle="modal" data-bs-target="#addContentContainer"><i class="fa-solid fa-pencil px-2"></i></button>')); //edit

        var level = node.getLevel();
        if (level != 1 && node.children.length == 0) {                        
            $title.prepend($('<i class="fa-solid fa-file px-2"></i>')); // file icon 
        }
        else //only add to folders
        {
            $title.append('<button class="btn" onclick="uploadFile(this)"><i class="fa-solid fa-upload px-2"></i></button>') // upload file
        }

        $title.append($('<button class="btn" onclick="deleteNode(this)"><i class="fa-solid fa-trash-can px-2"></i></button>')); //delete
        $title.append($('<button class="btn" onclick="duplicateNode(this)"><i class="fa-solid fa-copy px-2"></i></button>')); //duplicate

        $title.children().not(".fa-file").hide();
    }
});

$('#tree1').on(
    'tree.select',
    function (event) {
        if (event.node) {
            var node = event.node;
            var allNodes = $('.jqtree_common.jqtree-title');
            var htmlNode = $('.jqtree_common [aria-label=' + node.name + ']');
            htmlNode.children().show();
            allNodes.not(htmlNode).children().not(".fa-file").hide();
        } else {
            var pNode = event.previous_node;
            var pHtmlNode = $('.jqtree_common [aria-label=' + pNode.name + ']');
            pHtmlNode.children().not(".fa-file").hide();
        }
    }
)
</script>
</body>
```

**Comment by mbraak** [comment]
I'll have a look

**Comment by mbraak** [comment]
I added a fix in pr #757. The pr is not merged yet.

It does require that you prepend the node with `children: []`; then it's recognized as an empty folder:

```js
$('#tree1').tree(
  'prependNode',
  {
    name: 'test',
    id: 99,
    children:[]
  }
);
```

**Comment by mbraak** [comment]
The fix is included in version 1.7.5
